code/parachain/frame/composable-maths/src/dex/mod.rs

```diff
@@ -27,3 +29,88 @@ where
 		self.into_iter().all(|weight| !weight.is_zero())
 	}
 }
+
+/// Checks if `a` and `b` are within an acceptable level of computaion error.
+///
+/// Currently this checks within `1/100_000`.
+///
+/// # Example
+///
+/// ```rust
+/// # use sp_runtime::Permill;
+/// # use composable_maths::dex::per_thing_acceptable_computation_error;
+/// let a = Permill::from_parts(100_000);
+/// let b = Permill::from_parts(100_010);
+/// let c = Permill::from_parts(99_990);
+///
+/// assert!(per_thing_acceptable_computation_error(a, b));
+/// assert!(per_thing_acceptable_computation_error(a, c));
+/// ```
+///
+/// ```rust
+/// # use sp_runtime::Permill;
+/// # use composable_maths::dex::per_thing_acceptable_computation_error;
+/// let a = Permill::from_parts(100_000);
+/// let b = Permill::from_parts(100_020);
+/// let c = Permill::from_parts(99_980);
+///
+/// assert!(!per_thing_acceptable_computation_error(a, b));
+/// assert!(!per_thing_acceptable_computation_error(a, c));
+/// ```
+// TODO(benluelo): Make `c` a param?
```

Comment:
Possibly `epsilon` as well though possibly not a good idea for it to be flexible.

Reply:
c is supposed to be epsilon here, i'll update it. My thought was that different contexts have different precision requirements; but until that comes up i agree that having it hardcoded is the best